rust/extns/numaflow-sqs/src/source.rs

```diff
@@ -0,0 +1,574 @@
+use std::collections::HashMap;
+use std::time::Duration;
+use tokio::sync::{mpsc, oneshot};
+use tokio::time::Instant;
+
+use aws_config::{meta::region::RegionProviderChain, BehaviorVersion};
+use aws_sdk_sqs::config::Region;
+use aws_sdk_sqs::types::{MessageSystemAttributeName, QueueAttributeName};
+use aws_sdk_sqs::{Client};
+use bytes::Bytes;
+use chrono::{DateTime, TimeZone, Utc};
+
+use crate::{Error, Result};
```

Comment:
Group imports in `std-external-crate` order https://rust-lang.github.io/rustfmt/?version=master&search=#StdExternalCrate%5C%3A
We try to follow this for new changes.

Reply:
I have added ```--config group_imports=StdExternalCrate``` to my Rust Rover rustfmt arguments. Should we add a rustfmt.toml?